lib/matplotlib/backends/backend_pdf.py

```diff
@@ -874,9 +874,11 @@ def createType1Descriptor(self, t1font, fontfile):
         return fontdescObject
 
     def _get_xobject_symbol_name(self, filename, symbol_name):
-        return "%s-%s" % (
+        Fx = self.fontName(filename)
```

Comment:
Can you find a better variable name? `Fx` is not clear to me. Additionally, the capital letter violates PEP-8 naming conventions.

Reply:
This name is used throughout the module for font names; if you want to change it it should be changed throughout the module, so I think this can happen at another time.